src/types/guild.ts

```diff
@@ -660,3 +666,40 @@ export interface EditGuildTemplate {
   /** description for the template (0-120 characters) */
   description?: string | null;
 }
+
+/** */
+export interface MembershipScreeningPayload {
+  /** when the fields were last updated */
+  version: string;
+  /** the steps in the screening form */
+  form_fields: MembershipScreeningFieldPayload[];
+  /** the server description shown in the screening form */
+  description: string | null;
+}
+
+/** */
+export interface MembershipScreeningFieldPayload {
+  /** the type of field */
+  field_type: MembershipScreeningFieldTypes;
+  /** the title of the field */
+  label: string;
+  /** the list of rules */
+  values?: string[];
+  /** whether the user has to fill out this field */
+  required: boolean;
+}
+
+/** */
+export type MembershipScreeningFieldTypes =
+  /** Server Rules */
+  "TERMS";
+
+/**  */
```

Comment:
?

Reply:
The "Document Membership Screening" PR is not yet merged, so the URLs are not available.